modules/dynamodb/src/main/scala/dynamodb/model.scala

```diff
@@ -51,7 +62,6 @@ object model {
     def ss(values: Set[String]): AttributeValue = AttributeValue.SS(values)
     def ss(values: String*): AttributeValue = AttributeValue.SS(values.toSet)
 
-    def n(value: String): AttributeValue = AttributeValue.N(value)
```

Comment:
Regarding these smart constructors, I would like to change those as it is not possible to build the `ns` as at runtime `ns(xs: Int*)` and `ns(xs: Double*)` have the same signature.

Reply:
True. At some point I want to explore having `AttributeValue` be just a tagless thing over circe Json, but it's not a rabbit hole I want to go in right now,